pkg/operator/ceph/cluster/controller.go

```diff
@@ -262,6 +320,61 @@ func (c *ClusterController) onAdd(obj interface{}) {
 // ************************************************************************************************
 // Update event functions
 // ************************************************************************************************
+func (c *ClusterController) onK8sNodeUpdate(oldObj, newObj interface{}) {
+	// skip forced resyncs
+	if reflect.DeepEqual(oldObj, newObj) {
```

Comment:
Could all these checks for node status be moved to k8sutil/nodes.go? Other providers will likely want to use this  as well.

Reply:
OK